views/class.html

```diff
@@ -2,20 +2,26 @@
   <div class="row">
     <h2>Class <span ng-bind="class_name"></span></h2>
     <hr />
-    <p class="description"><span future-ng-bind="class_description">This is a dummy class!</span></p>
-    <p>Owner: <b><span future-ng-bind="class_owner">mrazo</span></b></p>
+    <p class="description"><span ng-bind="class_description"></span></p>
+    <p>Owner: <b><span ng-bind="class_owner"></span></b></p>
```

Comment:
I've realized that we can just do `{{ class_owner }}` instead of `ng-bind` which is a lot less readable. Also I changed to `{{ class.long_name }}` so it's more user friendly.